docs/designs/conceptual-versioning.md

```diff
@@ -0,0 +1,395 @@
+# Conceptual versioning - Phase 1
+
+This document specifies Docfx vnext conceptual versioning phase 1 design.
+
+## 1 Scope
+
+### 1.1 In scope
+
+#### For content writer
+
+1. Support expressing version constrains as [moniker range](#21-moniker-range)
+
+2. Support moniker zone markdown syntax.
+
+3. Support publish two file publish with same *sitePath* but different version, where the appropriate file is selected by url with query `view={moniker}`.
+
+4. All files will be built in one ground, which will bring a better performance.
+
+#### For content reader
+
+1. Support viewing page with specific version by query `?view={moniker}`
+
+2. Support showing Toc with different version.
+
+3. [Empty page](#42-empty-page) should be handled.
+
+### 1.2 Out of scope
+
+1. Not support specific version reference, including link and xref.
```

Comment:
>Not support specific version reference, including link and xref. [](start = 3, length = 64)

Is this caused by lacking of DHS fallback logic? If so, add some explanation about this.

Reply:
this is caused by the lacking of version info in resovled result.

---
In reply to: [227281807](https://github.com/dotnet/docfx/pull/3522#discussion_r227281807) [](ancestors = 227281807)